lasagne/tests/layers/test_special.py

```diff
@@ -170,43 +170,135 @@ def test_transform_errors():
 
 
 def test_transform_downsample():
+    import lasagne
+    downsample = (0.7, 2.3)
```

Comment:
Ah, no, I see, that was from an old commit of Søren, where the indentation was wrong. Then it's just a case of "not your business" again :)